shapediver-cli.ts

```diff
@@ -186,6 +187,113 @@ yargs(process.argv.slice(2))
             );
         },
     )
+    .command(
+        "notify-users",
+        "Notify our users about updates, maintenance, etc.",
+        (yargs) => 
+        {
+            yargs
+                .options({
+                    p: {
+                        alias: "plan name",
```

Comment:
@luka712 these aliases should not have spaces, please use `-` instead of space, otherwise it won't work on the CLI. Example why: 

```
axl@frodo:~/Arbeit/ShapeDiver/dev/serverless/TypeScriptSdkExample-Basic$ ./shapediver-cli.ts notify-users
shapediver-cli.ts notify-users

Notify our users about updates, maintenance, etc.

Optionen:
      --help                      Hilfe anzeigen                       [boolean]
      --version                   Version anzeigen                     [boolean]
  -p, --plan name                 Name of subscribed chargebee plan.
                                                         [string] [erforderlich]
  -f, --features                  Features of user.  If array, concat with ','.
                                  example: admin, exports, imports      [string]
  -o, --has organization          If users should be in organization. Can be one
                                  of 'y', 'n' or empty. If 'y' user must be in
                                  organization.
                                  If 'n' filters users without organization.
                                  Empty does nothing.                   [string]
  -r, --organization role         The organization role. Checks for user in
                                  organization role or in roles. If array,
                                  concat with ','.  example: owner, user[string]
  -d, --dry run                   If dry run, notifactions are not sent. Just
                                  returns and prints list of users which are
                                  fetched. Use 'y' for dry run.        [boolean]
  -h, --notification href         Link for notification. Can be used to link
                                  release notes etc...                  [string]
  -t, --notification type         The notification type. [string] [erforderlich]
  -n, --notification description  The description of a notification
                                                         [string] [erforderlich]

Fehlende Argumente: p, t, n
```